microsoft-edge/hosting/webview2/Tooling/WebDriver.md

```diff
@@ -0,0 +1,117 @@
+
+![alt text](..\images\WebDriver\Nuget.PNG)
+
+6. Use **OpenQA.Selenium.Edge** by adding the following using statement:```using OpenQA.Selenium.Edge;``` to the top of the file.
+
+```C#
+using OpenQA.Selenium.Edge;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+```
+
+## Step 4: Setup WebDriver
+
+1. First, create the **EdgeOptions** object, by copying the code below. The **EdgeOptions** object takes in two parameters:
+    1. **is_legacy** is set to false, indicating you are using the new Chromium-powered Microsoft Edge Browser.
+    2. The **"webview2"** string specifies that the **WebDriver** is driving **WebView2**.
+
+
+```C#
+static void Main(string[] args)
+{
+    // EdgeOptions() requires using OpenQA.Selenium.Edge
+    // Construct EdgeOptions with is_legacy = false and the string "webview2"
+    EdgeOptions edgeOptions = new EdgeOptions(false, "webview2");
+```
+
+2. Next, set **edgeOptions BinaryLocation** to the filepath of your WebView2 project's executable, create a string to store the path to **EdgeDriver's** executable, and create a string to store the name of the **EdgeDriver**. Use these two strings to construct the **EdgeDriverService** object as shown below. Finally, create the **EdgeDriver** object using **EdgeDriverService** and **EdgeOptions**.
+
+You can copy and paste the following code underneath **edgeOptions**. Make sure to specify the correct file paths to your project's executable and the edgedriver's executable on your machine.
+```C#      
+    //Set the BinaryLocation to the filepath of the WebView2API Sample's executable
+    edgeOptions.BinaryLocation = @"C:\path\to\your\webview2\project.exe";
+
+    //Set msedgedriverDir to the filepath of the MS Edge Driver you downloaded
+    string msedgedriverDir = @"C:\path\to\your\msededriver.exe\";
+
+    //Set msedgedriverExe to the name of the Edge Driver. By default it is:
+    string msedgedriverExe = @"msedgedriver.exe";
+
+    // Construct EdgeDriverService with is_legacy = false  
+    EdgeDriverService service = EdgeDriverService.CreateDefaultService(msedgedriverDir, msedgedriverExe, false);
+
+    EdgeDriver e = new EdgeDriver(service, edgeOptions);
+```
+3. Now, the **EdgeDriver** is ready to automate the **WebView** in your project. For example, if you are using the **WebView2API Sample**, you can **Navigate** to http://microsoft.com by caling ```e.Url = @"http://www.microsoft.com";```. You can watch **WebDriver** drive **WebView2** by setting a breakpoint on this line and running the project.
+
+```C#
+    //The following will Navigate the WebView2API Sample from bing.com to microsoft.com
+    e.Url = @"http://www.microsoft.com";
+
+    //This exits the edge driver
+    e.Quit();
+}
+```
+![alt text](..\images\WebDriver\Microsoft.PNG)
+
+Congratulations! You have successfully connected the WebDriver to a WebView2 project and driven WebView2.. 
```

Comment:
I would change this to: "You have successfully automated a WebView2 project and driven WebView2 using Selenium and Microsoft Edge Driver."